src/main/java/se/diabol/jenkins/pipeline/PipelinePagination.java

```diff
@@ -0,0 +1,118 @@
+/*
+This file is part of Delivery Pipeline Plugin.
+
+Delivery Pipeline Plugin is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+Delivery Pipeline Plugin is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with Delivery Pipeline Plugin.
+If not, see <http://www.gnu.org/licenses/>.
+*/
+package se.diabol.jenkins.pipeline;
+
+import hudson.model.Hudson;
+
+
+public class PipelinePagination {
+	private int totalCount = 0;
+	private int currentPage = 1;
+	private int prevPage = 0;
+	private int nextPage = 0;
+	private int totalPage = 0;
+	private int count = 0;
+	private int pageSize = 10;
+	private final String moveLink;
+	private String rootUrl = "";
+
+	
+	public PipelinePagination(int currentPage, int totalCount, int pagingSize, String moveLink) {
+		this.currentPage = currentPage;
+		this.totalCount = totalCount;
+		this.pageSize = pagingSize == 0 ? 10 : pagingSize;
+		this.moveLink = moveLink;
+		try {
+			rootUrl = Hudson.getInstance().getRootUrlFromRequest();
+		} catch (Exception e) {
+
+		}
+	}
+
+	
+	public String getTag() {		
+		setPageNation();
+		StringBuilder sb = new StringBuilder();
+		//sb.append("<table summary='page navigation' class='Nnavi'>\n").append("<tr>\n");
+		if (prevPage + 1 > 10) {
+			setPrePage(sb);
+		} else {
+			//setBlankSpan(sb);
```

Comment:
@pskumar448 could you please explain the intention of the commented out lines? We should not have any commented out code, if it isn't relevant, please just remove those lines.